Polyhedron/demo/Polyhedron/Scene_c3t3_item.cpp

```diff
@@ -273,9 +273,9 @@ struct Scene_c3t3_item_priv {
   {
     init_default_values();
     tet_Slider = new QSlider(Qt::Horizontal);
-    tet_Slider->setValue(100);
     tet_Slider->setMinimum(0);
     tet_Slider->setMaximum(100);
+    tet_Slider->setValue(100);
```

Comment:
Whoah, that one was a nasty bug! Previously the `setValue(100)` was called before `setMaximum(100)`, and the default maximum is... 99, as far as I know!